File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/qute-example/java/src/test/java/org/acme/qute/ItemsResourceTest.java
package org.acme.qute;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.containsString;

import org.junit.jupiter.api.Test;

import io.quarkus.test.junit.QuarkusTest;

@QuarkusTest
public class ItemsResourceTest {

    @Test
    public void testEndpoint() {
        given()
                .when().get("/qute/items")
                .then()
                .statusCode(200)
                .body(containsString("Apple:"), containsString("<del>30</del> <strong>27.0</strong>"));
    }

}

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/qute-example/kotlin/src/test/kotlin/org/acme/qute/ItemsResourceTest.kt
package org.acme.qute

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.hamcrest.CoreMatchers.containsString
import org.junit.jupiter.api.Test

@QuarkusTest
class ItemsResourceTest {
    @Test
    fun testEndpoint() {
        given()
                .`when`().get("/qute/items")
                .then()
                .statusCode(200)
                .body(containsString("Apple:"), containsString("<del>30</del> <strong>27.0</strong>"))
    }
}